### DISCUSSION TITLE
CORS

### DISCUSSION THREAD

**Discussion by JassiSingh08** [AuroPick/epic-free-games#24] [Q&A] [answered]
Hi, I am using this package. The only thing is that I do not want to have a backend server and using this package on my client side gives CORS ERROR. is there a solution for it? Can I use this inside my client and not get a CORS ERROR? 
Thanks

**Comment by AuroPick** [comment]
Hi @JassiSingh08 

I am gonna look at this in my free times

**Reply by AuroPick** [reply, marked as answer]
Hi again @JassiSingh08 

I couldn't find a way to bypass the CORS error. But it seems to work in Next.js
